site/irc.txt

```diff
@@ -0,0 +1,56 @@
+KISS IRC
+________________________________________________________________________________
+
+IRC was originally in the #kisslinux channel on Freenode, but was moved
+to Libera following the hostile behaviour on the part of Andrew Lee (see
+#/blog/20210512a).
+
+You can connect via Libera's webchat at https://web.libera.chat/#kisslinux.
+
+
+[1.0] Index
+________________________________________________________________________________
+
+- Public Logs                                                              [2.0]
+- IRC Etiquette                                                            [3.0]
+
+
+[2.0] Public Logs
+________________________________________________________________________________
+
+The IRC channel is publicly logged by a few of the channel denizens. Here are a
+list of public logs:
+
+- https://libera.irclog.whitequark.org/kisslinux/2021-05-29
+- (@phoebos) https://gemini.ctrl-c.club/~phoebos/logs/
+- (@cem) https://irclogs.carbslinux.org/kisslinux/
+- (@acheam) https://irclogs.armaanb.net/kisslinux/
+
+Unfortunately, the above sources don't contain any logs from before
+#kisslinux switched to Libera. For an archive of previous logs from
+Freenode, see https://freenode.logbot.info/kisslinux/.
```

Comment:
Note to future selves: this part will have to be updated at the end of June once we have the old logs.